src/platforms/react-native/manual-setup/expo.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Expo
+description: "Learn how to set up Expo managed project with Sentry React Native SDK."
+---
+
+<Alert level="info" title="Experimental Support">
+
+The Expo support is currently experimental and available since version `5.16.0-alpha.1` and Expo SDK 50.
```

Comment:
```suggestion
Expo support is currently experimental and available since Sentry React Native SDK version `5.16.0-alpha.1` and Expo SDK version 50.
```